lib/Doctrine/ORM/AbstractQuery.php

```diff
@@ -410,16 +412,24 @@ public function processParameterValue($value)
             return $value;
         }
 
-        if (is_object($value) && $this->_em->getMetadataFactory()->hasMetadataFor(ClassUtils::getClass($value))) {
+        if ($value instanceof Mapping\ClassMetadata) {
+            return $value->name;
+        }
+
+        if ( ! is_object($value)) {
```

Comment:
```suggestion
        if (! is_object($value)) {
```

Reply:
Take a look around the code: the old Doctrine standard (which is still being used in this branch) uses a space before the negation operator.

Reply:
Yeah, our new CS is only for `master`.